src/metaxy/metadata_store/base.py

```diff
@@ -185,12 +185,12 @@ def resolve_update(
         Args:
             feature: Feature class to resolve updates for
             samples: Pre-computed DataFrame with ID columns
-                and `PROVENANCE_BY_FIELD_COL` column. When provided, `MetadataStore` skips upstream loading, joining,
+                and `[[metaxy.models.constants.]]` column. When provided, `MetadataStore` skips upstream loading, joining,
```

Comment:
TODO fix this

I have a fix locally but cannot push

```
 git push
ERROR: user:1694964:geoHeil
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Anyways:

```
Must have `[[metaxy.models.constants.METAXY_PROVENANCE_BY_FIELD]]`
```

is not correctly interpreted from mkdocs